lib/utils.js

```diff
@@ -86,21 +86,34 @@ function patchProperties(obj, properties) {
 
 function patchEventTargetMethods(obj) {
   var addDelegate = obj.addEventListener;
-  obj.addEventListener = function (eventName, fn) {
-    fn._bound = fn._bound || {};
-    arguments[1] = fn._bound[eventName] = zone.bind(fn);
+  obj.addEventListener = function (eventName, handler) {
+    var fn;
+    
+    if (handler.handleEvent) {
+      // Have to pass in 'handler' reference as an argument here, otherwise it gets clobbered in
+      // IE9 by the arguments[1] assignment on line 106.
+      fn = (function(handler) {
+        handler.handleEvent.apply(handler, arguments);
+      }).bind(this, handler);
```

Comment:
Yes, unfortunately. It seems that closing over an argument and then subsequently reassigning `arguments[i]` in that scope breaks in IE9. That's why I had to use `bind`.